### DISCUSSION TITLE
Pixi 7 typescript warning on appendChild

### DISCUSSION THREAD

**Discussion by tmetzger** [pixijs/pixijs#9027] [Q&A] [answered]
Hey folks,

It seems like the most basic, "hello world" code, but in my typescript project this:
```
const app = new Application({ antialias: true});
document.body.appendChild(app.view);
```

causes a warning, specifically:

Argument of type 'ICanvas' is not assignable to parameter of type 'Node'.
  Type 'ICanvas' is missing the following properties from type 'Node': baseURI, childNodes, firstChild, isConnected, and 42 more.ts(2345)

The code still works fine and creates the graphics on the web page. Anyone know what's going on here?

Tom

**Comment by SuperSodaSea** [comment, marked as answer]
You need to specify the type like this, see #8820:
```ts
const app = new Application<HTMLCanvasElement>(...);
```

**Comment by tmetzger** [comment]
Oh! That's so simple. I'm surprised I couldn't find the original answer, I've been looking for a couple of hours.

Thanks for your quick response!
